app/controllers/attendees_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AttendeesController < ApplicationController
+  before_action :set_attendee, only: [:show, :edit, :update, :destroy]
+  before_action :set_event
```

Comment:
It is good approach to add `before_action` for parent object on top, so that it will run first.